src/interface/news/index.js

```diff
@@ -10,14 +10,14 @@ const News = () => (
   <>
     <DocumentTitle /> {/* prettiest is if the Home page has no title at all */}
 
-    <div className="flex">
+    <div className="flex flex-news">
       <div className="flex-main">
         <h1 id="news-top"><Trans>New stuff</Trans></h1>
       </div>
-      <div className="flex-sub" style={{ paddingTop: 30 }}>
+      <div className="flex-sub flex-sub-news">
         <small>More news?</small>
         <span style={{ fontSize: 18, marginLeft: 10 }}>
-          <TwitterIcon colored /> <a href="https://twitter.com/WoWAnalyzer">Follow @WoWAnalyzer</a>
+          <TwitterIcon colored /> <a href="https://twitter.com/WoWAnalyzer" style={{fontSize: '16px'}}>Follow @WoWAnalyzer</a>
```

Comment:
```suggestion
          <TwitterIcon colored /> <a href="https://twitter.com/WoWAnalyzer" style={{ fontSize: 16 }}>Follow @WoWAnalyzer</a>
```

In React it's best practice to use an integer for such properties. I don't remember if React will actually throwing a warning for this though.

Will this change the size of the text?

Reply:
> In React it's best practice to use an integer for such properties. I don't remember if React will actually throwing a warning for this though.
> 
> Will this change the size of the text?

Yes it will. Currently I I saw it sitting at 18px, which as you get to smaller devices, the link starts to wrap. 
I thought the better solution to this would be having it just being 16px all the time as even on desktop that still looks good to me! 
Happy to put this in a media query instead if you'd rather 18px at larger resolutions.

Reply:
16px sounds good